src/spectrecoff-cli/commands/Prompt.fs

```diff
@@ -84,6 +84,7 @@ type PromptDocumentation() =
             BI [
                 P "chooseFrom = (choices: string list) (question: string) -> string"
                 P "chooseMultipleFrom = (choices: string list) (question: string) -> string list"
+                P "chooseMultipleFromWith = (options: MultiSelectionPromptOptions) (choices: string list) (question: string) -> string list"
```

Comment:
Maybe add a sentence describing the new option?

Reply:
Done